dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/BroadcastClusterInvoker.java

```diff
@@ -89,12 +89,14 @@ public Result doInvoke(final Invocation invocation, List<Invoker<T>> invokers, L
         }
 
         if (exception != null) {
-            if (failIndex == failThresholdIndex) {
-                logger.debug(
+            if (logger.isDebugEnabled()) {
+                if (failIndex == failThresholdIndex) {
```

Comment:
Adjust the order of the two if judgments, so that the judgment condition of `isDebugEnabled() ` follows `logger.debug()`.

Reply:
done, thanks